pages/_error.tsx

```diff
@@ -0,0 +1,52 @@
+// CSS
+import styles from '../styles/Error.module.scss';
+// Components
+import NavBar from '../components/NavBar';
+import Button, { ButtonType } from '../components/Button';
+
+export default function ErrorPage({ 
+  statusCode 
+}:{
+  statusCode: any
+}) {
+  const returnTo = '/';
+
+  return (
+    <div className={styles.canvas}>
+      <NavBar currentPage='Home' />
+      <div className={styles.page}>
+        <div className={styles.container}>
+          <h1 className={styles.heading}>
+          Oops!
+          </h1>
+          <p className={styles.body}>
+            {statusCode
+              ? `A ${statusCode} error occurred on server`
+              : 'An error occurred on client'}
+          </p>
+          {/* Return to home page */}
+          <div>
+            <Button 
+              type={ButtonType.LIGHT}
+              href={returnTo}
```

Comment:
```suggestion
              href={returnTo}
              prefetch={true}
```
Double check thats the right prop, but the only place the user will be navigating from this page is home or the back button so it probably makes sense to prefetch the home page here, can you test it and if it doesnt incur any major wait or load time enable the prefetching?

Reply:
Doesn't seem to make a big difference in timing either way, let me know if you want it there or not. There is a pretty jarring white screen before the 404 loads, but I don't remember if it's always done that or not